### ISSUE TITLE
IDE - readthedocs keyword endpoint for hovers in VSCode Extension

### ISSUE THREAD

**Issue by Revilotom** [conjure-cp/conjure#412]
We could have some routes like:

```
conjure.readthedocs.io/keywords/<theKeyword>
```

That return information about the keyword in markdown format.

**Comment by ozgurakgun** [comment]
I am looking into this, but turns out I cannot build the documentation locally any more. Can anyone else? It seems to work fine on readthedocs.

/cc @ott2 @fraser-dunlop

**Comment by ott2** [comment]
``make docs`` still seems to work for me; this might be related to specific Sphinx versions?
``sphinxmanual.cls: sphinxmanual 2016/10/12 v1.5``

There were some non-fatal errors (reasonable PDF is still produced):
```
conjure/docs/demonstrations.rst:17: ERROR: Unexpected indentation.
conjure/docs/essence.rst:998: ERROR: Unexpected indentation.
conjure/docs/essence.rst:999: WARNING: Block quote ends without a blank line; unexpected unindent.
```
which I'll take a look at.

**Comment by ott2** [comment]
Looks like the branch is protected currently; my fixes for the three RST errors above await resolution of the current discussion.  Not sure this will fix @ozgurakgun being unable to build docs, though.

**Comment by ozgurakgun** [comment]
@Revilotom - this seems easy enough. I added a `docs/keywords/subsetEq.rst` file and it appeared here: https://conjure.readthedocs.io/en/latest/keywords/subsetEq.html

So if you generate links pointing to https://conjure.readthedocs.io/en/latest/keywords/X.html we can start populating those.

**Comment by Revilotom** [comment]
Awesome!

**Comment by ott2** [comment]
Documentation fixes mentioned above are in commit 12d2d2aa941b82ae5be4fbdb9094b0e2be2a08c3

@ozgurakgun are you still unable to build docs?
